haystack/nodes/file_classifier/file_type.py

```diff
@@ -1,40 +1,72 @@
+from multiprocessing.sharedctypes import Value
 from typing import List, Union
 from pathlib import Path
 from haystack.nodes.base import BaseComponent
 
 
+DEFAULT_TYPES = ["txt", "pdf", "md", "docx", "html"]
+
+
 class FileTypeClassifier(BaseComponent):
     """
     Route files in an Indexing Pipeline to corresponding file converters.
     """
-    outgoing_edges = 5
+    outgoing_edges = 10
+
+    def __init__(self, supported_types: List[str] = DEFAULT_TYPES):
+        """
+        Node that sends out files on a different output edge depending on their extension.
+
+        :param supported_types: the file types that this node can distinguish.
+            Note that it's limited to a maximum of 10 outgoing edges, which 
+            correspond each to a file extension. Such extension are, by default
+            `txt`, `pdf`, `md`, `docx`, `html`. Lists containing more than 10
+            elements will not be allowed. Lists with duplicate elements will 
+            also be rejected.
+        """
+        if len(supported_types) > 5:
```

Comment:
should be 10 instead of 5 if I am not mistaken as the docstring also says "Lists containing more than 10 elements will not be allowed"